Sources/NIO/ServerSocket.swift

```diff
@@ -66,24 +66,18 @@
     /// - throws: An `IOError` if the operation failed.
     func accept(setNonBlocking: Bool = false) throws -> Socket? {
         return try withUnsafeFileDescriptor { fd in
-            var acceptAddr = sockaddr_in()
-            var addrSize = socklen_t(MemoryLayout<sockaddr_in>.size)
-
-            let result = try withUnsafeMutablePointer(to: &acceptAddr) { (ptr) throws -> CInt? in
-                try ptr.withMemoryRebound(to: sockaddr.self, capacity: 1) { ptr in
-                    #if os(Linux)
-                    let flags: Int32
-                    if setNonBlocking {
-                        flags = Linux.SOCK_NONBLOCK
-                    } else {
-                        flags = 0
-                    }
-                    return try Linux.accept4(descriptor: fd, addr: ptr, len: &addrSize, flags: flags)
-                    #else
-                    return try Posix.accept(descriptor: fd, addr: ptr, len: &addrSize)
-                    #endif
-                }
+            var zero = socklen_t(0)
+            #if os(Linux)
+            let flags: Int32
+            if setNonBlocking {
+                flags = Linux.SOCK_NONBLOCK
+            } else {
+                flags = 0
```

Comment:
Nit: For simple ifs like this I'm a fan of ternaries, but not sure what the NIO standard is

Reply:
@Davidde94 that code is unchanged, just reindented.

Reply:
I saw - just a note, feel free to merge :)